solr/core/src/java/org/apache/solr/core/XmlConfigFile.java

```diff
@@ -165,7 +152,6 @@ public XmlConfigFile(
       db.setErrorHandler(xmllog);
       try {
         doc = db.parse(is);
```

Comment:
In this file, I suggested you could tweak it to use SafeXMLParsing to both reduce code but moreover to make the "Safe"-ness more obvious.  Did that turn out to be hard or just don't want to bother?  (either is fine)

Reply:
Thanks for the reminder @dsmiley. I missed that part. I'll do the changes.